### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Minor code comment grammar fixes.
- Removes unneeded EOF code comment.
- Combines properties that could be nested.
- Adjusts property order of CSS in cf-layout to match cfgov-refresh.
- Changes color variable to match hex code format used elsewhere.

### CODE REVIEW

File: src/cf-forms/src/atoms/text-input.less [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .a-text-input {
     .u-webfont-regular();
 
-    // Browser Reset
```
Comment (Scotchester) [inline src/cf-forms/src/atoms/text-input.less]: This is not a sentence, or even a phrase that implies a full sentence :) If you want to be consistent with the periods, rephrase.

File: src/cf-forms/src/atoms/select.less [hunk 2]
```diff
@@ -53,7 +53,7 @@
 }
 
 
-// TODO Add modernizr to CF so this works
+// TODO: Add modernizr to CF so this works
```
Comment (Scotchester) [inline src/cf-forms/src/atoms/select.less]: Missed adding a period here.

File: src/cf-layout/src/cf-layout.less [hunk 3]
```diff
@@ -545,29 +545,29 @@
...
     &:before {
-        content: "";
         display: block;
         height: 1px;
-        margin-bottom: unit(@grid_gutter-width / @base-font-size-px, em);
-        background-color: @grid_column__top-divider;
-        position: static;
         width: 100%;
         margin-left: auto !important;
+        margin-bottom: unit(@grid_gutter-width / @base-font-size-px, em);
```
Comment (Scotchester) [inline src/cf-layout/src/cf-layout.less]: Bottom should come before left, following the order of shorthand dimension declarations (top, right, bottom, left).

Comment (Scotchester) [inline src/cf-layout/src/cf-layout.less]: Add some spaces inside the parentheses while you're here.

Comment (anselmbradford) [inline src/cf-layout/src/cf-layout.less]: Fixed in https://github.com/cfpb/capital-framework/pull/499/commits/05326ae435338ff88371acd83e8079c902759ed3

File: src/cf-forms/src/atoms/select.less (hunk 2 above)
Comment (anselmbradford) [inline src/cf-forms/src/atoms/select.less]: Fixed in https://github.com/cfpb/capital-framework/pull/499/commits/05326ae435338ff88371acd83e8079c902759ed3

File: src/cf-forms/src/atoms/text-input.less (hunk 1 above)
Comment (anselmbradford) [inline src/cf-forms/src/atoms/text-input.less]: Fixed in https://github.com/cfpb/capital-framework/pull/499/commits/2d399cde7207aca5584be365852f4c815b685303

**Comment by jimmynotjim** [comment]
@anselmbradford I haven't had a chance to review, but make sure you include CHANGELOG entries for all the components you're editing. Looks like most will be a `[PATCH]`

File: src/cf-core/src/cf-vars.less [hunk 4]
```diff
@@ -12,62 +12,62 @@
...
 
 
 // Sizing variables
 
-@base-font-size-px:             16px;
-@base-line-height-px:           22px;
-@base-line-height:              unit(@base-line-height-px / @base-font-size-px);
+@base-font-size-px:      16px;
+@base-line-height-px:    22px;
+@base-line-height:       unit(@base-line-height-px / @base-font-size-px);
```
Comment (jimmynotjim) [inline src/cf-core/src/cf-vars.less]: Should prob add spaces within the parens while you're here.

**Comment by Scotchester** [comment]
Actually, I don't think this needs any change log entries. There are no visible updates, even in the docs. It's all comments and code formatting.

File: src/cf-core/src/cf-vars.less (hunk 4 above)
Comment (anselmbradford) [inline src/cf-core/src/cf-vars.less]: Fixed in https://github.com/cfpb/capital-framework/pull/499/commits/61606fba9e694fbe43965695937f89318c0f7e16

**Comment by jimmynotjim** [comment]
I guess folks aren't really looking at the source of the installed files so that makes sense.

**Pull request merged**